contracts/pol/CarbonPOL.sol

```diff
@@ -33,13 +36,17 @@ contract CarbonPOL is ICarbonPOL, Upgradeable, ReentrancyGuardUpgradeable, Utils
     // token to initial price mapping
     mapping(Token token => Price initialPrice) private _initialPrice;
 
+    // initial and current eth sale amount - for ETH->BNT trades
+    EthSaleAmount private _ethSaleAmount;
+
     // upgrade forward-compatibility storage gap
-    uint256[MAX_GAP - 3] private __gap;
+    uint256[MAX_GAP - 4] private __gap;
 
     /**
      * @dev used to initialize the implementation
      */
-    constructor() {
+    constructor(Token bntInit) {
+        _bnt = bntInit;
```

Comment:
Not super important, but I'd consider checking that `bntInit` (`initBnt`?) isn't 0x0